src/Slider/slider.css

```diff
@@ -60,6 +60,7 @@
 
         user-select:    none;
         cursor:         pointer;
+        -webkit-tap-highlight-color: transparent;
```

Comment:
Please make sure that indentation is kept consistent

Reply:
Done